### PR TITLE
switch to cloud

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: spec/query_spec.rb [hunk 1]
```diff
@@ -447,20 +451,14 @@ def test_method
...
       @refs << client.query { create @test_class, data: { z: 1 } }[:ref]
       @refs << client.query { create @test_class, data: { z: 1 } }[:ref]
     end
 
     it 'performs distinct' do
-      set = Fauna::Query.match(@test_over_z)
+      set = Fauna::Query.match(@test_by_z)
       distinct = Fauna::Query.distinct(set)
 
       expect(get_set_data(set)).to eq([0, 1, 1])
```
Comment (marrony) [inline spec/query_spec.rb]: I saw travis-ci is breaking on this assertion, why did you change `test_over_z` to `test_by_z`?

Comment (sprsquish) [inline spec/query_spec.rb]: I was trying to see if I could get the test to pass without cloud running its mappers (they're off while the migration happens). That doesn't seem to be happening, though. So I've put this off until the migration is complete.

Comment (marrony) [inline spec/query_spec.rb]: Are the migration done? Can we revert (remove) this last commit?

**Comment by sprsquish** [comment]
removed the last commit. this wont pass Travis until the migration completes though

**Comment by marrony** [comment]
When you say it won't pass it's because the indices won't get created until the migration ends?

I'm saying this because I've run the tests locally point go cloud.faunadb.com and it hangs forever

**Comment by sprsquish** [comment]
The test waits for the index to be built. Cloud's mappers are turned off right now so the indexes will never be built. I have to leave the mappers off until the migration is complete.

**Comment by marrony** [comment]
Since the tests are passing locally I will merge this PR

**Pull request merged**